packages/modal/src/types.ts

```diff
@@ -10,12 +10,13 @@ export type ModalContentAnimations = {
  * Defines options for modal behavior.
  */
 export type ModalOptions = {
-    defaultState?: "open" | "close", // Whether the modal should be open by default or not (default is close)
+    defaultState?: string, // Whether the modal should be open by default or not (default is close)
```

Comment:
It’s important to keep the type as "open" | "close" here, as defaultState can only have these two values. It's essential to maintain this specific typing at this level.